tests/block/test_get_block.py

```diff
@@ -0,0 +1,269 @@
+"""
+Provide tests for command line interface's get block by its identifier command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+EXISTING_BLOCK_IDENTIFIER = '4a7897650db9863aca34874778e6c5802f86c3df0e22b39cfea730bc83654357' \
+                            '037a422f8ef51ac85a9bc61d2484bd0f37be10cfc861588c41dc6f1bbfd92cde'
+
+
+def test_get_block():
+    """
+    Case: get a block by identifier.
+    Expect: block data is returned, batches are presented, header signature is block identifier.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'block',
+        'get',
+        '--id',
+        EXISTING_BLOCK_IDENTIFIER,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    block = json.loads(result.output).get('result')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert EXISTING_BLOCK_IDENTIFIER == block.get('header_signature')
+    assert len(block.get('batches')) > 0
+
+
+def test_get_block_invalid_identifier():
+    """
+    Case: get a block by invalid identifier.
+    Expect: the following block is invalid error message.
```

Comment:
`the following block is an invalid error message.`

Reply:
@Alladin9393, done.